docs/index.md

```diff
@@ -24,10 +26,18 @@ resource "logdna_view" "http500" {
 }
 ```
 
+## Pre-requirements and considerations
+Before using Terraform for creating resources in LogDNA, review the following notes:
+- Verify Terraform is installed. The minimum supported version is 0.12.0
```

Comment:
```suggestion
- Verify Terraform is installed. The minimum supported version is 0.12.0.
```

Reply:
Any reason this was resolved and not merged? Each of these statements in the list are sentences, so they should all end in periods.